src/util/jwtTokenGenerator.js

```diff
@@ -6,7 +6,6 @@ function base64url(source) {
 
   // Remove padding equal characters
   encodedSource = encodedSource.replace(/=+$/, '');
```

Comment:
It is the most optimal regex we can have for scanning the padded '='. Already tried possessive quantifiers and they are not efficient as this one